src/base/components/Tabs.js

```diff
@@ -17,10 +18,20 @@ import {
 import { findDOMNode } from 'react-dom';
 import EditExternal from 'CoreEditor/containers/EditExternal';
 const changeUrlToUrlForEdit = require('./utils/change_url_to_url_for_edit');
+import { connect } from 'react-redux';
+import { loadFeed } from '../../base/actions/actions';
+
+const feedURL  = 'https://muhammadumair0.github.io/feed/';
 
 const HEADER_PADDING = 15; // variable set in CSS
```

Comment:
No styles in JS. Delete

Reply:
I didnt added this

Reply:
Yes, i know, ,missed that one before. Please delete

Reply:
It was added by Michael Bilenko (2 years ago)

Reply:
ok

Reply:
removed

Reply:
Please be sure there no "HEADER_PADDING" in the code anywhere.

Reply:
ok